targets/kde

```diff
@@ -7,9 +7,13 @@ DESCRIPTION='Installs a minimal KDE desktop environment. (Approx. 600MB)'
 HOSTBIN='startkde'
 CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
-
+$TARGET=kde-plasma-desktop
```

Comment:
This won't work, should be TARGET='kde-plasma-desktop' Also try to use quotes when declaring a string.